src/Jeffijoe.MessageFormat.MetadataGenerator/Plural/Parsing/InvalidCharacterException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Jeffijoe.MessageFormat.MetadataGenerator.Plural.Parsing
+{
+    public class InvalidCharacterException : ArgumentException
```

Comment:
.NET has an exception for failing to parse: [FormatException](https://docs.microsoft.com/en-us/dotnet/api/system.formatexception?view=net-5.0).

Reply:
Forgot about that one for some reason, thanks